src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Models/RuleQuery.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Altinn.Platform.Authorization.Models
+{
+    /// <summary>
+    /// Queries for a list of already delegated rules from the supplied offeredby for the supplied app. If coveredby has any key roles, those party ids can be included in the query to have the 3.0 PIP lookup those as well. If offeredby is a sub unit, parenty party id can be supplied to include rules given from that party as well.
```

Comment:
This is a very long line. Could you wrap it once or twice? I personally try to ensure that no line is longer than 120 characters.

Reply:
I broke down the summary line into three shorter lines.